include/eve/function/lcm.hpp

```diff
@@ -43,8 +43,8 @@ namespace eve
   //!
   //!@warning
   //!    `p` and `n` can be of any [real values](@ref eve::value) type, but when the types are not integral
-  //!    the least common multiple is defined only if `p` and `n` element are [flint](@ref eve::is_flint). If it is not the
-  //!    case the corresponding result will be Nan.
+  //!    the least common multiple is defined only if `p` and `n` element are [flint](@ref eve::is_flint). If any
+  //!    is not flint the result is UB.
```

Comment:
We don't pay the doc per character: "If any of the arguments is flint, then the behavior is undefined."